### PR TITLE
manager: only sync label when label key exists

### PR DESCRIPTION
**Pull request by ZiMengSheng**
### V. Checklist

- [ ] I have written necessary docs and comments
- [ ] I have added necessary unit tests and integration tests
- [ ] All checks passed in `make test`

### CODE REVIEW

File: pkg/slo-controller/noderesource/plugins/gpudeviceresource/plugin.go [hunk 1]
```diff
@@ -183,13 +183,17 @@ func (p *Plugin) calculate(node *corev1.Node, device *schedulingv1alpha1.Device)
 	klog.V(5).InfoS("calculate gpu resources", "node", node.Name, "resources", gpuResources)
 
 	// calculate labels about gpu driver and model
-	if device.Labels != nil {
+	updatedLabels := map[string]string{}
+	if gpuModel, ok := device.Labels[extension.LabelGPUModel]; ok {
```
Comment (saintube) [inline pkg/slo-controller/noderesource/plugins/gpudeviceresource/plugin.go:187]: Does the controller need to handle the case where the node labels should be removed?

Comment (ZiMengSheng) [reply pkg/slo-controller/noderesource/plugins/gpudeviceresource/plugin.go:187]: No, doesn't need to.

**Review by FillZpp — APPROVED** [review]
/lgtm

**Pull request merged**